ui/src/main/webapp/lib/redux-polling/index.js

```diff
@@ -11,7 +14,7 @@ const sleep = (time) => new Promise((resolve) => {
 })
 
 // async actions
-export const poll = (id, asyncAction, { interval = 5000 } = {}) => (...args) => async (dispatch, getState) => {
+export const poll = (id, asyncAction, { interval = 1 } = {}) => (...args) => async (dispatch, getState) => {
```

Comment:
Woops, good catch. Left this in here while debugging, I'll change it back to 5000ms. Polling the backend 1000 times a second probably isn't ideal.